### ISSUE TITLE
add air density and potential temperature to export state

### ISSUE THREAD

**Issue by DeniseWorthen** [NOAA-EMC/ufsatm#122]
When cplflx = true, the air density and potential temperature should be added to the FV3 export state. They should be "height_lowest" values. 

These variables are needed by CICE5/CICE6; currently they are being calculated either in the NEMS mediator or in the CICE cap using temp_height_lowest, pres_height_lowest and spec_humid_height_lowest.

**Comment by DeniseWorthen** [comment]
This change would simplify the CICE6 cap, since calculations done in the cap to provide these fields could be removed. @junwang-noaa is it possible to pass these variables from FV3?

**Comment by junwang-noaa** [comment]
@DeniseWorthen We only need these two variables on the lowest model layer, correct?

**Comment by DeniseWorthen** [comment]
That is correct. In the CICE cap, we calculate the potential Temperature using the pres_height_lowest and the inst_temp_height_lowest. We calculate air density using pres_height_lowest, specific_humidity_lowest and temperature lowest. So the lowest level pTem and air density at lowest level would be consistent.
